unit-tests/run-unit-tests.py

```diff
@@ -476,6 +480,8 @@ def test_wrapper( test, configuration = None ):
         test.run_test( configuration = configuration, log_path = log_path )
     except FileNotFoundError:
         log.e( log.red + test.name + log.reset + ':', str(e) + configuration_str( configuration, prefix = ' ' ) )
+    except subprocess.TimeoutExpired:
+        log.e(log.red + test.name + log.reset + ':', configuration_str(configuration, suffix=' ') + 'timed out')

```

Comment:
add `after X seconds`

Reply:
X will always be equal to the timeout parameter, seems a bit unnecessary

Reply:
We may choose to have the timeout configurable per test (actually, that's a good next task!).
But this is for the user -- even if it repeats (which it shouldn't for many tests), when you see it failed because of a timeout, your question shouldn't be "what is the timeout?!"

Reply:
The time the test took is always printed, even if it timed out. So you will see:
test took X seconds
test-... timed out

Reply:
Well, one is debug (hidden unless you look at the log) and one is an error (what shows up on the pipeline page). But good enough.